src/controllers/auth.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require('jsonwebtoken');
+
+module.exports = (req, res, next)=>{
+  const {token} = req.cookies;
```

Comment:
What if `req.cookies` is `undefined` (The user has no cookies)? This would cause an error. Make it right.

Reply:
this is just the first step